src/main/java/com/greedobank/reports/controller/HealthCheckController.java

```diff
@@ -0,0 +1,19 @@
+package com.greedobank.reports.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HealthCheckController {
+    @Autowired
+    private Environment environment;
```

Comment:
Are you sure mix of @Autowired and constructor-based injectection is what you want?:)